### ISSUE TITLE
Question about ref vcf adding new variants

### ISSUE THREAD

**Issue by orisnas** [shilab/STI#13]
From reading the code and training pipeline, it seems that STICI requires a fixed SNP order (ID) because the model input shape and  embeddings depend on the order of variants.If the reference panel is updated with new SNPs (i.e., the number of variants changes), do we need to re-train the model with new VCF? Or is it possible to fine‑tune the existing model after expanding the SNP set?

**Comment by ErfanMowlaei** [comment]
The way it works is that it will do a right join (reference variants as the right side), so the model as it is will predict whatever variants were present in the reference given the shared variants among the training and the test set.

```do we need to re-train the model with new VCF? Or is it possible to fine‑tune the existing model after expanding the SNP set?``` > Fine-tuning is not implemented there and is generally tricky. I would suggest a fresh training with the right set of variants in that case.
